schemasheets/examine_schemasheet.py

```diff
@@ -0,0 +1,7 @@
+from schemasheets.schemasheet_datamodel import SchemaSheet
```

Comment:
Can you document this?

Reply:
I'm just going to delete that script. It was for learning by reverse engineering.

Reply:
I'm deleting this script.It was for learning by reverse engineering.